docs/9.0/guides/formulas/hyperformula-integration.md

```diff
@@ -4,14 +4,14 @@ metaTitle: HyperFormula integration - Guide - Handsontable Documentation
 permalink: /9.0/hyperformula-integration
```

Comment:
This page is deprecated? It's removed from sidebar.js

Reply:
AFAIK, this file should be removed however our migration guide links to it.
In the next version, it is removed. There is only updated `formula-calculation.md`.

@budnix, if I had a good memory, you worked about this. Am I right with the explanation above?

Reply:
Hmm, I don't remember working on this task 🤔